docs/user-guide/authenticating-with-client-certificates.md

```diff
@@ -0,0 +1,106 @@
+# Authenticating with client certificates
+
+You can perform authentication with client certificates.
+
+
+:::note
+There is a limitation with respect to performing authentication using Z Secure Services (ZSS) with ACF2 systems. If you are using ACF2, and are using Zowe v2.14 or later, use the recommended the internal API ML mapper.
+
+When providing credentials during a login request with a client certificate on the same login request, the login credentials take precedence and the client certificate is ignored.
+:::
+
+## How the Gateway resolves authentication 
+
+When sending a request to a service with a client certificate, the Gateway performs the following process to resolve authentication:
+
+* The client calls the service endpoint through the API ML Gateway with the client certificate.
+* The client certificate and private key are checked as a valid TLS client certificate against the trusted CAs of the Gateway.
+* The public part of the provided client certificate is checked against SAF. SAF subsequently returns a user ID that owns this certificate. As of Zowe version 2.14, the API for API ML can be provided if the internal API ML mapper is enabled. Alternatively, you can use Z Secure Services (ZSS) to provide this API for API ML.
+
+:::note**Notes:**
+* Currently ZSS is the default API that provides this mapping between the public part of the client certificate and SAF user Id, however, the recommended method is to use the internal API ML mapper.
+* For information about the internal API ML mapper, see [Enabling the internal API ML mapper](#enabling-the-internal-api-ml-mapper)described in this article.
+* For information about ZSS, see the section Zowe runtime in the [Zowe server-side installation overview](./install-zos).
+:::
+
+* The Gateway then performs the login of the mapped user and provides valid authentication to the southbound service.
+
+When sending a request to the login endpoint with a client certificate, the Gateway performs the following process to exchange the client certificate for an authentication token:
+
+* The client calls the API ML Gateway login endpoint with the client certificate.
+* The client certificate and private key are checked as a valid TLS client certificate against the trusted CAs of the Gateway.
+* The public part of the provided client certificate is checked against SAF. SAF subsequently returns a user ID that owns this certificate. As of Zowe release 2.14, the internal API ML mapper can provide this API for API ML if enabled in the zowe.yaml file. Alternatively, ZSS can provide this API for API ML, with the noted exception when using ACF2.
```

Comment:
maybe a link to the section with the instructions to enable the internal mapper? Or perhaps a table of contents in the beginning would help